js/shutterbug-support.ts

```diff
@@ -19,3 +21,37 @@ function beforeSnapshotHandler() {
     }
   });
 }
+
+export function takeSnapshot(selectorOrDomElement: string | HTMLElement) {
+  return new Promise((resolve, reject) => {
+    Shutterbug.snapshot({
+      selector: selectorOrDomElement,
+      done: (snapshotUrl: string) => {
+        resolve(snapshotUrl);
+      },
+      fail: (jqXHR: any, textStatus: string, errorThrown: any) => {
+        console.error("Shutterbug request failed", textStatus, errorThrown);
+        reject(errorThrown);
+      }
+    });
+  });
+}
+
+export function takePlanetViewSnapshot() {
+  return takeSnapshot(`#${PLANET_VIEW_CANVAS_ID}`);
```

Comment:
Just curious why you interpolate this since it is already a string?

Reply:
There's `#` in front of it. I guess it melts with `$`. So, it's `"#" + PLANET_VIEW_CANVAS_ID`.

Reply:
Oops. I missed the `#` - makes sense now.